packages/edit-site/src/components/header/index.js

```diff
@@ -79,64 +84,107 @@ export default function Header( {
 	const displayBlockToolbar =
 		! isLargeViewport || deviceType !== 'Desktop' || hasFixedToolbar;
 
+	// Track mouse location.
+	const [ mouseLoc, setMouseLoc ] = useState( { x: 0, y: 0 } );
+	useEffect( () => {
+		const setMouseLocFromEvent = ( event ) =>
+			setMouseLoc( { x: event.clientX, y: event.clientY } );
```

Comment:
`Non-blocking`:

Nitpicky comment. Whenever I've seen mouse x and y described in libraries like RXJs and in documentation like [w3 schools](https://www.w3schools.com/JSREF/event_clientx.asp) or [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent), the prevailing convention has been to use the term`coordinates` shortened to `coords` instead of `location` shortened to `loc`.

Reply:
`mouseCoords` instead of `mouseLoc` ?  Im happy going with something like that.